llvm/test/CodeGen/X86/MachineSink-Issue98477.ll

```diff
@@ -0,0 +1,36 @@
+; RUN: llc < %s
```

Comment:
Despite the fact that this test is checking whether llc crashes, I would prefer using FileCheck + UTC (Update Test Check) to check against the output just like other codegen tests, for the sake of uniformity.

Reply:
Updated the test.